### PR TITLE
8626 task: button component styles

### PR DESCRIPTION
**Pull request by creido-welly**
Link to designs: https://www.sketch.com/s/fdfb45f3-3e86-49d2-9f06-332acc86024c

* Updated styles inline with recent changes to wellcome.org
* Component specific CSS variables are scoped component styles
* Upgraded Storybook to latest version

buttons

### CODE REVIEW

File: src/styles/20-tools/_mixins/_link.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
+@mixin link-base {
```
Comment (creido-welly) [inline src/styles/20-tools/_mixins/_link.scss:1]: File is a combination of the following from corporate-components:

Placeholders from [extends > typography](https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/20-tools/_extends/_typography.scss) repurposed as mixins and `%anchor-` prefixes in names replaced with `link-`

Mixins from [mixins > typography](https://github.com/wellcometrust/corporate-components/blob/develop/src/assets/styles/20-tools/_mixins/_typography.scss)

File: src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -29,7 +27,7 @@ export default {
   argTypes: {
     onClick: { action: 'clicked' },
     variant: {
-      options: ['primary', 'secondary', 'ghost', 'link', 'unstyled'],
```
Comment (stuartwilkes) [inline src/components/Button/Button.stories.tsx:32]: We're no longer being haunted by the ghost button? This is definitely a better name...

**Review by stuartwilkes — APPROVED** [review]
Looks good 👻➡️🚪

**Comment by creido-welly** [comment]
tnx @stuartwilkes

**Pull request merged**